TheMovieDb/Components/CoordinatorProtocol.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  CoordinatorProtocl.swift
+//  TheMovieDb
+//
+//  Created by Juan David Torres on 04/11/21.
+//
+
+import Foundation
+import UIKit
+protocol Coordinator {
+  var children: [Coordinator] {get set}
+  var navigationController: UINavigationController {get set}
```

Comment:
```suggestion
  var children: [Coordinator] { get set }
  var navigationController: UINavigationController { get set }
```

spaces after each bracket